code/mapping/ext_modules/mapping_common/shape.py

```
@@ -1,13 +1,14 @@
 from dataclasses import dataclass
 from typing import Optional
-
 import rospy
-
+from typing import List, Optional
```

Ruff (0.8.2)

Redefinition of unused `Optional` from line 2

Remove definition: `Optional`

(F811)